examples/README.md

```diff
@@ -1,4 +1,68 @@
-# Doorman Integration Examples
+# Integration Examples
 
 - [Python / Flask](python/): A Web UI interacts with Auth0 and a Flask API
 
+# Configuration Examples
+
+## Superusers with Doorman tags
```

Comment:
The word "superuser" is quite a personal choice. That's what *I* call it over in Symbols because that project is based on Django and they use that word to refer to users who have unlimited access. 
The word is quite powerful and since, in the case of Symbols, that membership thing gets used in relation to security. 

Perhaps a better example, would be something harmless. Like "catlovers" or "contentmoderators". 
Because, effectively, the ask is that Doorman is the shared vault that maintains this curated list of people who can admin my system.